ui/app/(ai)/analyst/(tools)/findings.ts

```diff
@@ -0,0 +1,28 @@
+import { tool } from "@langchain/core/tools";
+
+import { getFindings, getMetadataInfo } from "@/actions/findings";
+import { getFindingsSchema, getMetadataInfoSchema } from "@/types/lighthouse";
+
+export const getFindingsTool = tool(
```

Comment:
Following up on this comment: https://github.com/prowler-cloud/prowler/pull/7508#discussion_r2052086824 

Could we please use the `/app` folder only for routing purposes?

Thank you!

Reply:
Moved all tools code to `/lib/lighthouse/tools/` directory